packages/@icon-magic/build/test/helpers/ideal-icon.ts

```diff
@@ -0,0 +1,135 @@
+import {
+  IconConfig,
+  FlavorTypeMap,
+  GenerateConfig,
+  BuildConfig,
+  AssetConfig,
+  FlavorConfig,
+  Icon,
+  BuildPlugin,
+  Asset
+} from '@icon-magic/icon-models';
+import { getNameFromPropCombo } from './utils';
+import * as path from 'path';
+
+const FIXTURES = path.resolve(__dirname, '..', '..', '..', 'test', 'fixtures');
+
+let sampleExt: FlavorTypeMap = {
+  svg: {
+    name: 'filled-a.svg',
```

Comment:
So the initial reason behind having each type map to an asset config was if we have to apply multiple plugins on each type and I thought we might need the flexibility in case a plugin modifies the name of the asset. I'm ok either way - this makes the config file more clear vs. if we have to infer it.